mne/viz/epochs.py

```diff
@@ -897,73 +899,62 @@ def plot_epochs_psd(epochs, fmin=0, fmax=np.inf, tmin=None, tmax=None,
         Either "full" or "length" (default). If "full", the PSD will
         be normalized by the sampling rate as well as the length of
         the signal (as in nitime).
-    %(picks_good_data)s
+    %(plot_psd_picks_good_data)s
     ax : instance of Axes | None
         Axes to plot into. If None, axes will be created.
-    color : str | tuple
-        A matplotlib-compatible color to use.
-    area_mode : str | None
-        Mode for plotting area. If 'std', the mean +/- 1 STD (across channels)
-        will be plotted. If 'range', the min and max (across channels) will be
-        plotted. Bad channels will be excluded from these calculations.
-        If None, no area will be plotted.
-    area_alpha : float
-        Alpha for the area.
-    dB : bool
-        If True, transform data to decibels.
+    %(plot_psd_color)s
+    %(plot_psd_xscale)s
+    %(plot_psd_area_mode)s
+    %(plot_psd_area_alpha)s
+    %(plot_psd_dB)s
+    %(plot_psd_estimate)s
+    %(show)s
     %(n_jobs)s
-    show : bool
-        Show figure if True.
+    %(plot_psd_average)s
+    %(plot_psd_line_alpha)s
+    %(plot_psd_spatial_colors)s
     %(verbose)s
 
     Returns
     -------
     fig : instance of Figure
-        Figure distributing one image per channel across sensor topography.
+        Figure with frequency spectra of the data channels.
     """
-    from .raw import _set_psd_plot_params, _convert_psds
+    # this chunk should be removed for 0.2
+    if average is False and spatial_colors is None:
+        spatial_colors = True
+    if spatial_colors is None:
+        spatial_colors = False
+        warn('spatial_colors defaults to False in 0.19 but will change to True'
+             ' in 0.2. Set it explicitly to avoid this warning.',
```

Comment:
0.20 (it's a version number as opposed to a decimal, so the truncation matters :) )